delfin/db/sqlalchemy/models.py

```diff
@@ -286,3 +287,112 @@ class FailedTask(BASE, DelfinBase):
     job_id = Column(String(36))
     deleted_at = Column(DateTime)
     deleted = Column(Boolean, default=False)
+
+
+class StorageHostInitiator(BASE, DelfinBase):
+    """Represents the storage host initiator attributes."""
+    __tablename__ = 'storage_host_initiators'
+    id = Column(String(36), primary_key=True)
+    storage_id = Column(String(36))
+    name = Column(String(255))
+    description = Column(String(255))
+    alias = Column(String(255))
+    wwn = Column(String(255))
+    status = Column(String(255))
+    native_storage_host_id = Column(String(255))
+    native_storage_host_initiator_id = Column(String(255))
+
+
+class StorageHost(BASE, DelfinBase):
+    """Represents the storage host attributes."""
+    __tablename__ = 'storage_hosts'
+    id = Column(String(36), primary_key=True)
+    storage_id = Column(String(36))
+    name = Column(String(255))
+    description = Column(String(255))
+    os_type = Column(String(255))
+    ip_address = Column(String(255))
+    status = Column(String(255))
+    native_storage_host_id = Column(String(255))
+
+
+class StorageHostGroup(BASE, DelfinBase):
+    """Represents the storage host group attributes."""
+    __tablename__ = 'storage_host_groups'
+    id = Column(String(36), primary_key=True)
+    storage_id = Column(String(36))
+    name = Column(String(255))
+    description = Column(String(255))
+    native_storage_host_group_id = Column(String(255))
+
+
+class PortGroup(BASE, DelfinBase):
+    """Represents the port group attributes."""
+    __tablename__ = 'port_groups'
+    id = Column(String(36), primary_key=True)
+    storage_id = Column(String(36))
+    name = Column(String(255))
+    description = Column(String(255))
+    native_port_group_id = Column(String(255))
+
+
+class VolumeGroup(BASE, DelfinBase):
+    """Represents the volume group attributes."""
+    __tablename__ = 'volume_groups'
+    id = Column(String(36), primary_key=True)
+    storage_id = Column(String(36))
+    name = Column(String(255))
+    description = Column(String(255))
+    native_volume_group_id = Column(String(255))
+
+
+class MaskingView(BASE, DelfinBase):
+    """Represents the masking view attributes."""
+    __tablename__ = 'masking_views'
+    id = Column(String(36), primary_key=True)
+    storage_id = Column(String(36))
+    name = Column(String(255))
+    description = Column(String(255))
+    native_storage_host_group_id = Column(String(255))
+    native_volume_group_id = Column(String(255))
+    native_port_group_id = Column(String(255))
+    native_storage_host_id = Column(String(255))
+    storage_host_initiators = Column(Text)
+    volumes = Column(Text)
+    ports = Column(Text)
```

Comment:
confused about the below defintion, can you explain it
```
    storage_host_initiators = Column(Text)
    volumes = Column(Text)
    ports = Column(Text)
```

Reply:
Provided some details on this at driver api side.